src/NuGet.Core/NuGet.Protocol/Plugins/PluginFile.cs

```diff
@@ -20,6 +20,22 @@ public sealed class PluginFile
         /// </summary>
         public Lazy<PluginFileState> State { get; }
 
+        /// <summary>
+        /// Is the plugin file, a dotnet tools plugin file?
+        /// </summary>
+        public bool IsDotnetToolsPlugin { get; }
```

Comment:
Is this class ever used outside of this assembly?
It seems to me that we never really needed it to the public. 

We can consider not making public changes.